Sources/Datadog/RUMMonitor.swift

```diff
@@ -593,9 +596,16 @@ public class RUMMonitor: DDRUMMonitor, RUMCommandSubscriber {
     // MARK: - RUMCommandSubscriber
 
     func process(command: RUMCommand) {
+        guard let scope = core.v1.scope(for: RUMFeature.self) else {
+            return
```

Comment:
(just checking) This `return` is unreachable, right? We do early-return in `RUMMonitor.initialize()` when `RUMFeature` is not registered, so it's not possible to run here.

Reply:
Yes, we do an early return in `RUMMonitor.initialize()` with appropriate console prints 👍